### ISSUE TITLE
Tracking issue for PR: predicates & inheritance

### ISSUE THREAD

**Issue by niomaster** [utwente-fmt/vercors#365]
See PR #270 for discussion

**Comment by niomaster** [comment]
I think we now (or soon) will have a better implementation via abstract predicate families, right @bobismijnnaam ?

**Comment by bobismijnnaam** [comment]
Yes, I'm starting on the prototype this week. When it is functional we can evaluated if we want to merge it into vercors (probably after some more work).
